tests/python/frontend/onnx/test_forward.py

```diff
@@ -5383,6 +5375,12 @@ def verify_eyelike(indata, dynamic=False):
     ]
 }
 
+flaky_target_skips = {
```

Comment:
Can this be removed now?

Reply:
Unfortunately not, bc it fails if cuda is enabled but passes without it. Here's the minor differences between the actual onnx val and tvm val for the `gridsample_nearest` test case on nvptx

```
actual = array([[[[0., 0., 2., 2.],
         [2., 2., 5., 0.]]]], dtype=float32), 
desired = array([[[[0., 0., 2., 3.],
         [3., 2., 5., 0.]]]], dtype=float32), 
```

This could be due to the small discrepancy in the implementations noted [here](https://github.com/apache/tvm/pull/13163#discussion_r1002119311). I think, the best is to track it in an issue for now.

For the record, I'm triggering the CI now to see it without the skip.

Reply:
So the CI passed and it's flaky comparing with my local result. I'll have to include the skip and make a note of it.

Reply:
Yeah interesting, on https://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH__DOUBLE.html#group__CUDA__MATH__DOUBLE_1g2316a104cfda8362208d52238181fbfb:

`__device__​ double nearbyint ( double  x )`
`Round argument x to an integer value in double precision floating-point format. Uses round to nearest rounding, with ties rounding to even.`

So it sounds like the CUDA rounding is fixed to round to even. This is odd because this is not a mode for the std::nearbyintf

Reply:
But yeah, sus, will be on the lookout for this,